cmd/enval.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"Adhara-Tech/check-my-setup/cmd/version"
+	"Adhara-Tech/check-my-setup/pkg/adapters"
+	"Adhara-Tech/check-my-setup/pkg/config"
+	"Adhara-Tech/check-my-setup/pkg/infra"
+	"Adhara-Tech/check-my-setup/pkg/manifestchecker"
+	"Adhara-Tech/check-my-setup/pkg/model"
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	name = "tchecker"
+)
+
+var cmd = &cobra.Command{
+	Use:  "tchecker",
+	Long: name,
+	RunE: executeCmd,
+}
+
+const (
+	manifestFlag = "manifest"
+)
+
+func main() {
+
+	cmd.Flags().String(manifestFlag, "", "path to the manifest file")
```

Comment:
the default value should be something like `.enval.yaml` or `.enval.manifest.yaml`? Or use the const `DefaultManifestFile` from the config package?